### PR TITLE
Add Klarna Badge and Express Checkout Button to Cart Sidebar

### PR DESCRIPTION
**Pull request by PickertJoe**
#### What does this PR do?
This PR adds the Klarna messaging badge and express checkout button link to the cart sidebar component. To ensure that our Klarna components have a single source of truth across our apps, `KlarnaCreditPromotionBadge` has been migrated into this library from Thor. Moving forward, we'll import these components in their respective front end apps and pass any relevant env vars to them as needed.

#### Screenshots
Screenshot from 2022-01-24 10-13-46
Screenshot from 2022-01-24 10-13-37

#### Relevant Tickets
https://app.shortcut.com/rockets/story/9591/customers-can-pay-via-klarna-express-checkout-from-bag

### CODE REVIEW

File: src/components/klarnaExpressCheckoutButton/KlarnaExpressCheckoutButton.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+
+class KlarnaCreditPromotionBadge extends React.Component {
```
Comment (manmartinez) [inline src/components/klarnaExpressCheckoutButton/KlarnaExpressCheckoutButton.js:3]: Should we rename this class to `KlarnaExpressCheckoutButton`?

File: src/components/klarnaExpressCheckoutButton/KlarnaExpressCheckoutButton.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    script.src = klarnaExpressCheckoutScriptSource
+    script.id = 'klarnaCheckoutButtonScript'
+    script.setAttribute('data-id', klarnaMerchantId)
+    script.setAttribute('data-environment', klarnaEnvironment)
+    script.async = true
+    document.head.appendChild(script)
+  }
+
+  componentWillUnmount () {
+    document.querySelector('#klarnaCheckoutButtonScript').remove()
```
Comment (manmartinez) [inline src/components/klarnaExpressCheckoutButton/KlarnaExpressCheckoutButton.js:23]: Would it make sense to store a reference to the `script` tag so we don't have to add an `id` and query the document?

```js
componentDidMount () {
  ...
  this.script = document.createElement('script')
  ...
}

componentWillUnmount () {
  this.script.remove()
}
```

**Review by manmartinez — APPROVED** [review]
Looks good, just had a minor comment :+1:

File: src/components/klarnaExpressCheckoutButton/KlarnaExpressCheckoutButton.js (hunk 1 above)
Comment (PickertJoe) [reply src/components/klarnaExpressCheckoutButton/KlarnaExpressCheckoutButton.js:3]: Ah yes, thank you! :sweat_smile:

**Pull request merged**